pkg/plugin/workloadattestor/linux/util.go

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright The SPIFFE Project & Scytale, Inc
```

Comment:
```suggestion
 * Copyright The SPIFFE Project & Scytale, Inc
 * Copyright (c) 2023 Cisco and/or its affiliates. All rights reserved.
```

Again, remove the MIT header and add this **if** the file has been changed